### DISCUSSION TITLE
Option to Disable / Hide the native Chrome horizontal tab bar.

### DISCUSSION THREAD

**Discussion by stefan123t** [Able-Bear-Software/tab-nodes-tree#74] [Ideas / Feature Request]
I am looking for an Option to Disable / Hide the native Chrome horizontal tab bar,
just like we can do away with in the Firefox tree-style-tabs using the userChrome settings.

**Comment by zeroarst** [comment]
Hey @stefan123t 
AFAIK, there's no way to do that in Chrome. If you find one, please let us know here. Thank you!"

**Comment by Biep** [comment]
What **is** possible (in Brave) is showing the tabs vertically, and then collapse them.  That will not completely hide them, but in landscape less screen real estate is consumed by it.
